nav2_regulated_pure_pursuit_controller/src/parameter_handler.cpp

```diff
@@ -308,6 +311,8 @@ ParameterHandler::updateParametersCallback(
         params_.use_cost_regulated_linear_velocity_scaling = parameter.as_bool();
       } else if (name == plugin_name_ + ".use_collision_detection") {
         params_.use_collision_detection = parameter.as_bool();
+      } else if (name == plugin_name_ + ".stateful") {
+        params_.stateful = parameter.as_bool();
```

Comment:
make sure to add this to the dynamic parameters unit test please

Reply:
I added it to the unit test